### ISSUE TITLE
Should /$type/<id> return basic node info, or all associations

### ISSUE THREAD

**Issue by cmungall** [monarch-initiative/biolink-api#168]
API is not consistent at the moment

## Option 1

 * `/$type/<id>` returns basic node properties (and potentially HAL-type links to other API calls)
 * `/$type/<id>/all` can be used to to fetch node + assocs if required

## Option 2

 * `/$type/<id>` returns node + assocs 
 * `/$type/<id>/info` returns basic node properties

**Comment by cmungall** [comment]
I favor option 1. Lends itself better to asyn calls to populate different parts of page etc

**Comment by kshefchek** [comment]
for /$type/<id>, it might be useful to return basic node properties and stats on associations, but think all associations should be a different function

**Comment by deepakunni3** [comment]
@kshefchek Do you mean something like this:

```
{  
   "taxon":{  
      "id":"NCBITaxon:9606",
      "label":"Homo sapiens"
   },
   "xrefs":null,
   "description":null,
   "categories":[  
      "gene",
      "sequence feature"
   ],
   "types":null,
   "synonyms":null,
   "deprecated":null,
   "replaced_by":null,
   "consider":null,
   "id":"HGNC:18603",
   "label":"COL25A1".
   "association_stats":{
        "genotype_associations":0,
        "phenotype_associations":10,
        "interaction_associations":0,
        "disease_associations":4,
        "homology_associations":10,
    }
}
```

**Comment by kshefchek** [comment]
yes exactly, we may also consider associations over homology (disease, phenotype) as well, but would require more computation.  There might be a trade off on performance and comprehensiveness.

**Comment by deepakunni3** [comment]
Even if its association stats, the number of queries to SciGraph and/or Solr remains the same. The difference is only in the way the results are parsed and sent back.
